vendor/diff-parsers-ext-parsewyc.patch

```diff
@@ -605,9 +605,9 @@
    | NEW ext_attributes mkrhs(class_longident)
        { Pexp_new($3), $2 }
    | LPAREN MODULE ext_attributes module_expr RPAREN
-       { Pexp_pack $4, $3 }
-   | LPAREN MODULE ext_attributes module_expr COLON package_core_type RPAREN
-       { Pexp_constraint (ghexp ~loc:$sloc (Pexp_pack $4), $6), $3 }
+       { Pexp_pack ($4, None), $3 }
+   | LPAREN MODULE ext_attributes module_expr COLON package_type RPAREN
+       { Pexp_pack ($4, Some $6), $3 }
 -  | LPAREN MODULE ext_attributes module_expr COLON error
 -      { unclosed "(" $loc($1) ")" $loc($6) }
```

Comment:
Why remove this rule ? (and the others below)

Reply:
All the error cases have been removed from the recovery parser because if we exit on error we don't recover (not done in this PR, what we see is the lines around the diff).
That was done by Frederic when he implemented it at first, and I hadn't had a look at it since, I don't know if we can modify the algorithm to work while keeping the error cases.